More categories (work on issue #60)

mozkeeler opened the pull request:
Actually, some of these seem a little short. Maybe I can find some more...
Also, I would like to try the google sets and see how they look.

mozkeeler committed:
start on having more categories

mozkeeler committed:
split up medical

monicachew commented on util/top_gaming_sites.txt:
```diff
@@ -0,0 +1,50 @@
+leagueoflegends.com
+gamefaqs.com
+king.com
+games.yahoo.com
```
This is problematic, it's going to get listed as yahoo.com

mozkeeler replied:
Hmmm - the whole way we handle domains is problematic in the same way. If someone manually blushes "games.yahoo.com", it will blushlist all of yahoo...
This may be a larger architectural issue we have to change. In the meantime, I'll remove these from these lists.

monicachew commented on util/top_medical_sites.txt:
```diff
@@ -0,0 +1,41 @@
+alumbo.com
+alz.org
+apa.org
+arthritis.about.com
```
same

monicachew commented:
I think this is a good start, modulo the TLD issues. Do you want me to wait til you add more?

mozkeeler committed:
remove some sites that won't work, add terms

mozkeeler commented:
Ok - I removed the sites that don't work and added some terms from google sets (which actually works pretty well, it seems). If this looks good, I'll regenerate blushlist.js and then we'll be good to go, I guess?

monicachew commented:
Sounds good to me.
